src/PKSim.Core/Snapshots/Mappers/ExtendedPropertyMapper.cs

```diff
@@ -17,12 +17,10 @@ public override Task<SnapshotExtendedProperty> MapToSnapshot(ModelExtendedProper
          {
             snapshot.ListOfValues = extendedProperty.ListOfValuesAsObjects.Any() ? extendedProperty.ListOfValuesAsObjects.ToList() : null;
             snapshot.Value = extendedProperty.ValueAsObject;
-            snapshot.Type = extendedProperty.Type;
             snapshot.Description = SnapshotValueFor(extendedProperty.Description);
             snapshot.Name = extendedProperty.Name;
-            snapshot.FullName = extendedProperty.FullName;
-            snapshot.DisplayName = extendedProperty.DisplayName;
```

Comment:
What's the difference between fullName and displayName?

Reply:
DisplayName returns FullName if not null, otherwise name. 
DisplayName is readonly